pkg/cmd/repo/create/create.go

```diff
@@ -248,10 +248,15 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 }
 
 func createRun(opts *CreateOptions) error {
+	cfg, err := opts.Config()
+	if err != nil {
+		return err
+	}
+	hostname, _ := cfg.Authentication().DefaultHost()
 	if opts.Interactive {
 		answer, err := opts.Prompter.Select("What would you like to do?", "", []string{
-			"Create a new repository on GitHub from scratch",
-			"Create a new repository on GitHub from a template repository",
+			"Create a new repository on " + hostname + " from scratch",
+			"Create a new repository on " + hostname + " from a template repository",
 			"Push an existing local repository to GitHub",
```

Comment:
```suggestion
			"Push an existing local repository to " + hostname,
```

Additionally, this will break tests, too.

Reply:
Which tests? All tests seem to be passing.

Reply:
https://github.com/cli/cli/pull/8664/files#diff-1383afb03d1730b7f23e2da529e535aa1fc1c5ac500fc802899dafa66fcbb61eR260

Reply:
Sorry, let me rephrase that:

> _"Making this change will require making additional changes in tests"_

https://github.com/cli/cli/blob/334592c69fcc5d3349705cdbede1fd0f5b9cc9f4/pkg/cmd/repo/create/create_test.go#L411

https://github.com/cli/cli/blob/334592c69fcc5d3349705cdbede1fd0f5b9cc9f4/pkg/cmd/repo/create/create_test.go#L477